common/src/main/java/org/apache/drill/common/util/function/CheckedConsumer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.common.util.function;
+
+import java.util.function.Consumer;
+
+@FunctionalInterface
+public interface CheckedConsumer<T, E extends Throwable> {
+  void accept(T t) throws E;
+
+  static <T> Consumer<T> throwingConsumerWrapper(
+    CheckedConsumer<T, Exception> throwingConsumer) {
+
+    return i -> {
+      try {
+        throwingConsumer.accept(i);
+      } catch (Exception ex) {
+        throw new RuntimeException(ex);
```

Comment:
Maybe it's better to use sneakyThrow to avoid wrapping exception in runtime one.
https://github.com/apache/drill/blob/a2533917b0bc387711060c739dfb9844896fe4be/common/src/main/java/org/apache/drill/common/util/function/CheckedFunction.java#L47
https://github.com/apache/drill/blob/a2533917b0bc387711060c739dfb9844896fe4be/common/src/main/java/org/apache/drill/common/exceptions/ErrorHelper.java#L193

Reply:
Ok. Sure. I will use it. Thanks.